graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -111,17 +112,17 @@
     private static final Logger LOG = LoggerFactory.getLogger(Indices.class);
     private static final String REOPENED_INDEX_SETTING = "graylog2_reopened";
 
+    public static final String ALL_INDEX_PREFIX = "*";
```

Comment:
This, together with https://github.com/Graylog2/graylog2-server/pull/3121/files#diff-1e045693d5a6863d0e42f1f49bc4dc90R326 leads us to read the information for all indices that contain an `_` character, which might be problematic.

Can we collect all the index prefixes instead and restrict the query to the indices that we should care about instead?